Tasks/app-store-release/app-store-release.ts

```diff
@@ -43,6 +43,27 @@ function findIpa(ipaPath: string): string {
     return paths[0];
 }
 
+async function isFastlaneInstalled(): Promise<boolean> {
+    tl.debug('Checking for previous fastlane version first...');
+    let gemRunner: ToolRunner = tl.tool(tl.which('gem', true));
```

Comment:
doesn't `tool` call `which` internally?  I think that's part of the convenience function